lib/r10k/module/forge.rb

```diff
@@ -89,7 +90,7 @@ def status
     if not self.exist?
       # The module is not installed
       return :absent
-    elsif not @metadata.exist?
+    elsif not File.exist?(@path + 'metadata.json')
```

Comment:
I'm still learning when Ruby-ism is correct and when it's over done, but the calling a method on the object result of an addition feels a little weird.